lib/tasks/sync_measurements_counter.rake

```diff
@@ -0,0 +1,6 @@
+desc "Sync measurements count on streams"
+task :sync_measurements_counter => [:environment] do
+  Stream.all.each do |stream|
```

Comment:
Please use something like [`find_each`](https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each) or something similar that batches the loop. Otherwise, Ruby will load into memory all records at once